README.md

```diff
@@ -57,6 +61,36 @@ config('something.which.does.not.exist');
 // ^^^ TypeScript will call you out on this one
 ```
 
+### Serialize your configuration with `config.toJSON()`
+You can serialize your entire configuration object to JSON using `config.toJSON()`.
+This is useful for logging or debugging purposes.
+
+```ts
+console.log(JSON.stringify(config.toJSON(), null, 2));
```

Comment:
when `toJSON()` is defined then `JSON.stingify()` will use that to get the serialisable value, so `console.log(JSON.stringify(config, null, 2))` would also work

Reply:
yep that's true - tests and docs were initially using just the `config` but added `.toJSON()` to show that the option is available. This also depends largely on whether we go with `.toJSON()` or something else

Reply:
since we're going with the `.toJSON()` I will update the docs and tests to just use `config`